src/elements/common/header/__tests__/Header.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { HeaderBase as Header } from '../Header';
+
+describe('elements/common/header/Header', () => {
+    const intl = {
+        formatMessage: jest.fn().mockImplementation(message => message.defaultMessage),
+    };
+
+    const getWrapper = props => shallow(<Header intl={intl} {...props} />);
+
+    test('renders Logo component when isHeaderLogoVisible is passed', () => {
+        const wrapper = getWrapper({ isHeaderLogoVisible: true });
+        expect(wrapper.find('Logo').exists()).toBe(true);
+    });
+
```

Comment:
Should we also add a test for the negative case above? (i.e. doesn't render logo when `isHeaderLogoVisible` is not passed or falsey)